extras/chat.revolt.RevoltDesktop.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
Is the desktop file removed from upstream? I can't find it there.

Reply:
nope its not removed so thats a bit weird

Reply:
Hmm, I can't find it in the official revolt upstream for some reason. I've found the commit in which it was removed (https://github.com/revoltchat/desktop/commit/f5d669e0dbed80bdc1d54e0417e1ef8039710cc4). I that case, this desktop file needs to be here :). Could all the files be moved into the root of the repo? There's not a lot of them and it would make it easier to see what this repo contains.